docs/sections/ci_features.md

```diff
@@ -0,0 +1,162 @@
+# CI features
+
+## Environments
+
+
+## List of executions
+
+
+## How matrix execution works 
+
+`Popper` supports a matrix syntax when using environment variables and execution environments.
+This means that Popper will execute all possible combinations between those elements returning
+a popper_status and its corresponding `.err` and `.out`' files for each one of them.
+
+To achieve this, you only need to add your environment variables and your execution environments
+as follows and `Popper` will do it for you automatically. 
+
+For example, if you were to add your first set of environment variables, you'd execute the following:
+
+`popper env-vars your-popper-pipe --add key1=val1 --add key2=val2`
+
+This will cause the following to be added to your `.popper.yml` file. 
+
+```
+vars:
+- key1: val1
+  key2: val2
+```
+
+So far, we only have one possible combinations so let's add more.
+You can use the same command to add more sets of environment variables to your 
+matrix execution.
+
+`popper env-vars your-popper-pipeline --add key3=val3 --add key3=val3`
+
+Which will result in the following:
+
+```
+vars:
+- key1: val1
+  key2: val2
+- key3: val3
+  key4: val4
+```
+
+This is how all the configuration of your pipeline should look like so far assuming you
+just made it using `popper init your-popper-pipeline`:
+
+```
+pipelines:
+  your-popper-pipeline:
+    envs:
+    - host
+    path: pipelines/pipe
+    requirements: {}
+    stages:
+    - setup
+    - run
+    - post-run
+    - validate
+    - teardown
+    vars:
+    - key1: val1
+      key2: val2
+    - key3: val3
+      key4: val4
+```
+
+Please notice that Popper automatically uses host as your default execution environment.
+Then, so far we have a matrix execution which consists in the combinations between host
+and your sets of environment variables. Let's add Debian 9 as an execution environment with the
+following command:
+
+`popper env your-popper-pipeline --add debian-9`
+
+###### NOTE: Docker is required in order to run your project on an execution environment different from host
```

Comment:
This can be a `> **NOTE**` instead of a 6-level header

Reply:
Sure, I made the change. @ivotron